tasks/main.yml

```diff
@@ -1,10 +1,53 @@
 # tasks file
 ---
+
 - name: include variables
   include_vars: "{{ item }}"
   with_first_found:
     - "_{{ ansible_distribution_release }}.yml"
     - "_{{ ansible_distribution | lower }}.yml"
     - _default.yml
-  tags:
-    - configuration
+
+- name: install | requirements
+  apt:
+    name: "{{ item }}"
+    state: "{{ apt_install_state | default('latest') }}"
+    cache_valid_time: "{{ apt_update_cache_valid_time | default(3600) }}"
+    update_cache: true
+  with_items:
+    - binutils
+    - git
+
+- name: install | pull amazon-efs-utils
+  git:
+    repo: 'https://github.com/aws/efs-utils'
+    dest: /tmp/efs-utils
+    version: "{{ aws_efs_utils_version }}"
+
+- name: install | build amazon-efs-utils
+  command: bash build-deb.sh || /bin/true
```

Comment:
consider using `shell` instead of `command`, e.g.:

```yaml
shell: "{{ efsutils_dest_dir }}/build-deb.sh"
```

Reply:
I got this "command-instead-of-shell" error in lint when I attempted to change that.